x/supply/internal/keeper/common_test.go

```diff
@@ -1,25 +1,18 @@
-package keeper
+package keeper_test
```

Comment:
I think we should be more explicit in the sense that these are integration tests. Can we rename it to `integration_test.go`

Reply:
definitely! I like this idea a lot

Reply:
seems like golang doesn't allow this:
```
can't load package: package github.com/cosmos/cosmos-sdk/x/supply/internal/keeper: found packages keeper (account.go) and integration (bank_test.go) in github.com/cosmos/cosmos-sdk/x/supply/internal/keeper
```

from [stackoverflow](https://stackoverflow.com/questions/20427890/does-it-make-sense-to-have-two-packages-in-the-same-directory)
```
The 2 packages must have the following names:

- NameOfDirectory.
- NameOfDirectory + _test.
The names of the files in the _test package must end with _test.go
```

So I think we should probably leave it as keeper_test. I would suggest adding a folder `integration` but these tests really are just testing the keeper's integration into the rest of the modules.

Reply:
Sorry, I meant the file lol